### DISCUSSION TITLE
Yumurta satıcısı sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#295] [Soru Cevap]
Eski zamanlarda bir yumurta tüccarı varmış. Yumurta toptancısına gidip bir miktar yumurta almak istemiş.
Toptancı ona ne kadar yumurta istediğini sormuş. O da cevaplamış:
– Hmm ne kadar yumurta istediğimi tam olarak bilmiyorum ama 100’den fazla olacağı kesin. Yumurtalarımı ikişer ikişer satarsam bana bir yumurta kalsın Yumurtalarımı üçer üçer satarsam gene bana bir yumurta kalsın. Yumurtalarımı dörder dörder satınca da bana bir yumurta kalsın. Beşer beşer altışar altışar yedişer yedişer sekizer sekizer dokuzar dokuzar onar onar satsam da hep bana bir yumurta kalsın Ama demiş onbirer onbirer satarsam bana yumurta kalmasına gerek yok! Toptancı başlamış düşünmeye. Sizce yumurtacı toptancıdan en az kaç yumurta almak istemiş?

**Comment by yigidi** [comment]
25201 buldum. Deneyerek yaptım.
